apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java

```diff
@@ -44,25 +40,57 @@
 /**
  * The <code>AgentClassLoader</code> represents a classloader, which is in charge of finding plugins and interceptors.
  */
-public class AgentClassLoader extends ClassLoader {
+public class AgentClassLoader extends URLClassLoader {
+    private static final ILog LOGGER;
+    /**
+     * Storage plug-in jar package url address
+     */
+    private static URL[] ARRAY_PLUGINS_URL;
```

Comment:
Could explain what is `ARRAY_PLUGINS_URL`? The name seems strange.

Reply:
It looks like `PLUGIN_JAR_URLS`?

Reply:
`ARRAY_PLUGINS_URL` is because the plugin url is an array type, it is not as clear as `PLUGIN_JAR_URLS`, I will adjust it here.